repoMiner.py

```diff
@@ -6,170 +6,193 @@
 # Here begins a list of potentially interesting repos to mine
 #
 # https://github.com/rust-lang/regex (lots of bug fixes for regex related code)
-# https://github.com/google/re2 
+# https://github.com/google/re2
+
 
 # Notes:
 # TODO: update these lists as time goes on, maybe add additional criteria for evolution since a lot are being pulled
 # TODO: Maybe use code churn methods to see average commit , or lines count, could also incop dmm params for additional analysis
-def plotAllEngine(mode,dataDict):
+def plotAllEngine(mode, dataDict):
     plt.figure()
     keylist = []
     for key in dataDict:
         plt.plot(dataDict[key].index, dataDict[key].values, marker="o")
         keylist.append(key)
     plt.xlabel("Year")
     plt.ylabel("Number of Items")
-    plt.title( mode + " Items Found Each Year")
+    plt.title(mode + " Items Found Each Year")
     plt.legend(keylist)
     plt.grid(True)
-    #plt.show()
+    # plt.show()
     plt.savefig(mode + "_EngTotal_line_graph.png")
-    
-def plotSingleEngine(totalItemsPerYear,evoItemsPerYear,maintainItemsPerYear,key):
+
+
+def plotSingleEngine(totalItemsPerYear, evoItemsPerYear, maintainItemsPerYear, key):
     plt.figure()
     plt.plot(totalItemsPerYear.index, totalItemsPerYear.values, marker="o")
-    plt.plot(evoItemsPerYear.index,evoItemsPerYear.values, marker="s")
-    plt.plot(maintainItemsPerYear.index,maintainItemsPerYear.values, marker="^")
+    plt.plot(evoItemsPerYear.index, evoItemsPerYear.values, marker="s")
+    plt.plot(maintainItemsPerYear.index, maintainItemsPerYear.values, marker="^")
     plt.xlabel("Year")
     plt.ylabel("Number of Items")
     plt.title(key.upper() + "- Number of Items Found Each Year")
     plt.legend(["All", "Evolution", "Maintenance"])
     plt.grid(True)
-    #plt.show()
-    plt.savefig(key+"_line_graph.png")
-    
-    
+    # plt.show()
+    plt.savefig(key + "_line_graph.png")
+
+
 def mineRepo():
-    repoDict = {"rust": "https://github.com/rust-lang/regex.git",
-                "re2":"https://github.com/google/re2.git",
-                "pcre" :"https://github.com/PCRE/pcre2.git"
-                }
+    repoDict = {
+        "rust": "https://github.com/rust-lang/regex.git",
+        "re2": "https://github.com/google/re2.git",
+        "pcre": "https://github.com/PCRE/pcre2.git",
+        "java": "https://github.com/openjdk/jdk",
```

Comment:
Were you able to get the pydriller to work with parsing this repo? Ive been trying on your branch for a bit now and its not done yet. I know we discussed taking this one away potentially but either way Im not sure if we want it in the program if it doesnt work currently